tests/general/default_config_test.go

```diff
@@ -110,7 +110,7 @@ func TestDefaultGatewayConfig(t *testing.T) {
 					"batch": nil,
 					"memory_limiter": map[string]any{
 						"check_interval": "2s",
-						"limit_mib":      "460",
+						"limit_mib":      460,
```

Comment:
This commit should be reverted if we are disabling the `confmap.strictlyTypedInput` feature gate.